fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1668,6 +1675,38 @@ dropFileStatement
 showSmallFilesStatement
     : SHOW FILE ((FROM | IN) catalog=qualifiedName)?
     ;
+// -------------------------------------------- Pipe Statement ---------------------------------------------------------
+
+pipeName:
+    qualifiedName
+    ;
+
+createPipeStatement
+    : CREATE PIPE (IF NOT EXISTS)? pipeName
+        properties?
+        AS insertStatement
+    ;
+
+dropPipeStatement
+    : DROP PIPE (IF EXISTS)? pipeName
+    ;
+
+alterPipeClause
+    : PAUSE | RESUME
```

Comment:
what happens if just send `PAUSE` to SR?

Reply:
```
(1064, "Getting syntax error at line 1, column 16. Detail message: Input '<EOF>' is not valid at this position.")
```